db/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM camptocamp/postgres:9.6
-
+# RUN rm -r /var/lib/postgresql/data/*
```

Comment:
This can be removed, the data from the base image (camptocamp/postgres) will always be empty.